index.js

```diff
@@ -13,6 +13,8 @@ const YAML = require('yaml');
 
 async function run(rootDir, options = {}) {
   let log = options.log || console.log;
+  let writeToFile = options.writeToFile || fs.writeFileSync;
+  let shouldFix = process.argv.includes('--fix');
```

Comment:
I would prefer to pass `fix: true/false` in as one of the `options`. That way we don't rely on the global `process.argv` state and this `run()` function stays reusable. It also means you don't have to use the `process.argv.push('--fix')` hack below 😉

Reply:
like this?
```suggestion
  let shouldFix = options.fix || process.argv.includes('--fix');
```

Reply:
I was thinking more like this:

```suggestion
  let shouldFix = options.fix ?? false;
```

and then in `bin/cli.js` we use `run(rootDir, { fix: process.argv.includes('--fix') })`